core/src/main/scala/org/locationtech/rasterframes/expressions/package.scala

```diff
@@ -68,6 +68,7 @@ package object expressions {
     registry.registerExpression[ExplodeTiles]("rf_explode_tiles")
     registry.registerExpression[GetCellType]("rf_cell_type")
     registry.registerExpression[SetCellType]("rf_convert_cell_type")
+    registry.registerExpression[SetNoDataValue]("rf_with_nodata")
```

Comment:
naming, this is otherwise referred to as `rf_with_no_data`.

Reply:
The names don't always match up, and the thought of changing it to `WithNoData` doesn't sit well with me when I look at the other class names. I'm not sure I'm a fan of `rf_convert_cell_type` or `rf_with_nodata`, but going and making everything precisely consistent seems like a rabbit hole. I'd like to leave it if it's OK with you.